### DISCUSSION TITLE
is there any group on discord , telegram , slack etc.. ?

### DISCUSSION THREAD

**Discussion by wausin** [orchidsoftware/platform#1714] [General]
.

**Comment by tabuna** [comment]
Hi @wausin. At the moment, we have 3 official telegram groups. 
Links are posted in the readme of the project. Also duplicated here:

### Telegram User Groups

* [Global Community](https://t.me/orchid_community)
* [Russian Community](https://t.me/orchid_russian_community)
* [Spanish Community](https://t.me/esLaravelOrchid)
